src/plugins/resolver/CMakeLists.txt

```diff
@@ -8,12 +8,17 @@ find_package(Threads)
 add_plugin(resolver
 	SOURCES
 		${SOURCES}
+	LINK_LIBRARIES
+		${CMAKE_THREAD_LIBS_INIT}
+		${CMAKE_REALTIME_LIBS_INIT}
```

Comment:
Would be better if this is in another commit (reverting my change). Not very important though.